specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json

```diff
@@ -473,6 +741,68 @@
         }
```

Comment:
Adding new properties to an existing resource in the same API version is considered a breaking change. Can you introduce a new API version?

Reply:
We are not making any changes to the existing domain service root resource and just adding a new nested resource. Do we still need to rev to a new API version?

Reply:
It looks like a diff bug. I was seeing an addition of 3 properties to the existing resource schema. Compared manually against what is committed. There's no need to introduce a new API version.